test/__snapshots__/Storyshoots.spec.js.snap

```diff
@@ -11255,10 +11255,10 @@ exports[`Storyshots Layout Auto 1`] = `
               className="base"
               style={
                 Object {
-                  "borderBottomWidth": 0,
+                  "borderBottomWidth": 24,
```

Comment:
This looks like the behavior changed, no?

Reply:
Oops, good catch, [this line](https://github.com/obartra/reflex/pull/300/files/4c0235c0d53e09d7fd1d87883f0ad23af6894ba8#diff-d13958befac094f9da077d89a15a314fR14) should be `"0 L/2"` instead of `"L L/2"`, I wonder why the screenshots didn't catch this, maybe it's below the fold?